aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -3,7 +3,9 @@
 module aptos_framework::fungible_asset {
     use aptos_framework::event;
     use aptos_framework::object::{Self, Object, ConstructorRef, DeleteRef};
+    use aptos_framework::aggregator_v2::{Self, Aggregator};
```

Comment:
nit: Sort imports alphabetically. We'll have a formatter that does this soon. For now you could do manually or use the VSCode plugin